Crashlytics/Crashlytics/Components/FIRCLSUserLogging.m

```diff
@@ -198,7 +198,8 @@ void FIRCLSUserLoggingCompactKVEntries(FIRCLSUserLoggingKVStorage *storage) {
     // but it's very uncommon to go down this path.
     NSArray *keys = [finalKVs allKeys];
 
-    FIRCLSSDKLogInfo("Truncating KV set, which is above max %d\n", maxCount);
+    FIRCLSSDKLogInfo("Truncating %lu keys from KV set, which is above max %d\n",
+                     ([finalKVs count] - maxCount), maxCount);
```

Comment:
Super Nit: We're going towards using the dot notation for things like count `[finalKVs count] => finalKVs.count`